tests/appsec/rasp/test_span_tags.py

```diff
@@ -68,5 +71,6 @@ def setup_sqli_span_tags(self):
         self.r = weblog.get("/rasp/sqli", params={"user_id": "' OR 1 = 1 --"})
 
     @missing_feature(library="dotnet")
+    @missing_feature(library="nodejs", reason="Not supported yet")
```

Comment:
If we are skipping these tests from manifest, do we need the method decorators?

Reply:
sqli, ssrf and lfi tests are into `Test_Mandatory_SpanTags` class, AFAIK we can't skip only sqli and lfi in the manifest file, in the same class, `ssrf` test should be executed.

Reply:
That's kinda annoying. Maybe the tests are badly organized and should be split between instrumentation points ? I don't see a reason why they're grouped like that, it makes versioning more difficult

Reply:
tbh, I don't have intention to change all rasp tests, at least in this PR that only pretends to enable some of them, to add other `@missing_feature`s. 
For example for blocking or stack traces, there are not enabled in this PR because is not implemented in the tracer, but if we start to organize these tests by type, like:
```
class Test_SSRF
  def test_span_tags:
  def test_stack_traces:
  def test_blocking:
```
We will continue having `@missing_feature`s for blocking and stack traces.